src/routes/news/containers/LocalNewsDetails.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+
+import * as React from 'react'
+import { connect } from 'react-redux'
+import { push } from 'redux-first-router'
+import { LocalNewsModel } from '@integreat-app/integreat-api-client'
+import type { TFunction } from 'react-i18next'
+import type { StateType } from '../../../modules/app/StateType'
+import Page from '../../../modules/common/components/Page'
+import NewsController from './../containers/NewsController'
+import ContentNotFoundError from '../../../modules/common/errors/ContentNotFoundError'
+import FailureSwitcher from '../../../modules/common/components/FailureSwitcher'
+
+type PropsType = {|
+  localNewsDetails: LocalNewsModel,
+  language: string,
+  city: string,
+  path: string,
+  t: TFunction
+|}
+
+export class LocalNewsDetailsPage extends React.PureComponent<PropsType> {
+  render () {
+    const { localNewsDetails, language, city, path } = this.props
+
+    if (!localNewsDetails) {
```

Comment:
localNewsDetails is an object, i think this check is not safe, since if the localNewsDetails is empty object so it will be !{} the result will be false, unless we are 100% guarantee it will return null, or undefined.

Reply:
This will be always null or the data object. so I believe it's safe to use it like this.

Reply:
That's fine